src/umpire/Umpire.hpp

```diff
@@ -122,6 +122,19 @@ bool pointer_overlaps(void* left, void* right);
  */
 bool pointer_contains(void* left, void* right);
 
+/*!
+ * \brief Check whether or not an Allocator is accessible from a given platform.
+ *
+ * This function describes which allocators should be accessible
+ * from which CAMP platforms. Information on platform/allocator
+ * accessibility can be found at
+ * <https://umpire.readthedocs.io/en/develop/features/allocator_accessibility.html>
+ *
+ *\param camp::Platform p  
+ *\param umpire::Allocator a 
+ */
+bool is_accessible(Platform p, Allocator a);
```

Comment:
Shouldn't this be:
```suggestion
bool is_accessible(camp::Platform p, Allocator a);
```

Reply:
I include the Platform.hpp file, so "Platform p" is really doing "camp::resources::Platform p" - I don't think it's needed, but I may be wrong.

Reply:
Oh.  I see.  The comment at line 133 says "camp::Platform".   Platform, umpire::Platform, and camp::Platform are going to be confusing.  I think Platform.hpp existed before camp came around.  My personal opinion is we should somehow deprecate Platform.hpp and instead use camp::Platform.

Reply:
Wait, maybe I don't lol let me look..

Reply:
We do use camp::Platform, but umpire::Platform is an alias.